docs/source/release-notes/1.3.0.rst

```diff
@@ -0,0 +1,59 @@
+1.3.0: 2019-01-25
+-----------------
+
+Features Added
+``````````````
+
+- Partial Checks support. We added the following:
+
+  - ``_RepoCommit.check_runs`` to retrieve the check runs for this commit
+
+  - ``_RepoCommit.check_suites`` to retrieve the check suites for this commit
+
+  - ``CheckRun.update`` to update the check run
+
+  - ``CheckSuite.check_runs`` to retrieve the check runs for this suite
+
+  - ``CheckRunOutput.annotations`` class to retrieve the annotations for a check run
+
+- Add ``_Repository.unignore`` method to unignore notifications from this repository for the user.
+
+- Add ``_Repository.unsubscribe`` method to unsubscribe the user to this repository's notifications.
+
+- Webhooks support. We added the following:
+
+  - ``_Organization.create_hook`` to create a hook on organization
+
+  - ``_Organization.hook`` to get a single hook on organization
+
+  - ``_Organization.hooks`` to return all hooks registered on organization
+
+  - ``OrganizationHook.delete`` to delete hook
+
+  - ``OrganizationHook.edit`` to edit hook
+
+  - ``OrganizationHook.ping`` to ping hook
+
+- A ``Project.retrieve_issue_from_content`` method was added to retrieve an Issue from the content url.
+
+- A ``Project.retrieve_pull_request_from_content`` method was added to retrieve a PullRequest from the content url.
+
+- We now support nested teams.
```

Comment:
I think other areas avoid talking about the project team and prefer active language, e.g.,

```rst
- Add support for webhooks in an organization by adding:

  - ``OrganizationHook`` class

  - ``create_hook``, ``hook``, and ``hooks`` methods to ``ShortOrganization`` and ``Organization`` classes

- Add support for Parent/Child teams via the ``hellicat-preview`` API preview type

- Add support for specifying merge commit titles when merging a Pull Request.
```